### PR TITLE
rbd: error out if import image format failed

### PR DESCRIPTION
**Pull request by wangzhengyong**
Signed-off-by: wangzhengyong@cmss.chinamobile.com

### CODE REVIEW

File: src/tools/rbd/action/Import.cc [hunk 1]
```diff
@@ -752,6 +752,10 @@ static int do_import(librbd::RBD &rbd, librados::IoCtx& io_ctx,
   } else {
     r = do_import_v2(fd, image, size, imgblklen, pc);
   }
+  if (r < 0) {
+    std::cerr << "rbd: failed to import image format" << std::endl;
+    goto err;
```
Comment (dillaman) [inline src/tools/rbd/action/Import.cc]: You would need to let this fall through to ensure the image is properly closed

File: src/tools/rbd/action/Import.cc [hunk 2]
```diff
@@ -752,6 +752,10 @@ static int do_import(librbd::RBD &rbd, librados::IoCtx& io_ctx,
   } else {
     r = do_import_v2(fd, image, size, imgblklen, pc);
   }
+  if (r < 0) {
+    std::cerr << "rbd: failed to import image format" << std::endl;
```
Comment (dillaman) [inline src/tools/rbd/action/Import.cc]: Nit: I would drop "format" at the end of the sentence

File: src/tools/rbd/action/Import.cc (hunk 1 above)
Comment (wangzhengyong) [inline src/tools/rbd/action/Import.cc]: i forget this...

File: src/tools/rbd/action/Import.cc [hunk 3]
```diff
@@ -752,6 +752,11 @@ static int do_import(librbd::RBD &rbd, librados::IoCtx& io_ctx,
   } else {
     r = do_import_v2(fd, image, size, imgblklen, pc);
   }
+  if (r < 0) {
+    std::cerr << "rbd: failed to import image" << std::endl;
+    image.close();
+    goto err;
```
Comment (dillaman) [inline src/tools/rbd/action/Import.cc]: Nit: still need to close the image

**Pull request merged**